modules/sdk/gradle-plugins-workspace/CHANGELOG.markdown

```diff
@@ -224,11 +224,12 @@ version 3.2.5.
 3.12.46.
 
 ## 1.10.2 - 2018-06-18
-- [LPS-74544]: Updated dependency to [Liferay Gradle Plugins] to version
+- [LPS-74544]: Update the dependency to [Liferay Gradle Plugins] to version
```

Comment:
If you take a look at the rest of the changelogs (just a couple of lines above this), you'll see that the format we use is as follows:

```
- [LPS-82534]: Update the [Liferay Gradle Plugins] dependency to version
 3.12.46.
```